### DISCUSSION TITLE
Import options?

### DISCUSSION THREAD

**Discussion by coraltwitcher** [cnr-isti-vclab/TagLab#36] [General]
Any ideas if you can import a data table (eg ID, Class, Attributes)? You can export these data (export annotations as data table), and it would be super useful to edit these in say excel/VBA for example and then import back in? At present data entry/editing is pretty tedious, requiring hopping between class/attribute/data table windows etc. Thanks!

**Comment by maxcorsini** [comment]
For the moment we do not think at this because the most properties are recomputed (perimeter, area, and so on). But the new possibility to add custom attributes, importing and exported table is perfectly reasonable. We will include such import in next release.

**Comment by coraltwitcher** [comment]
Thank you very much!
Paul
